--- a/zsh/.zshrc
+++ b/zsh/.zshrc
@@ -12,7 +12,14 @@
 # -----------------------------
 # Completions
 # -----------------------------
+zmodload zsh/complist
 autoload -Uz compinit && compinit

+zstyle ':completion:*' menu select
+zstyle ':completion:*' matcher-list 'm:{a-z}={A-Z}'
+zstyle ':completion:*:default' list-colors ${(s.:.)LS_COLORS}
+
+bindkey -M menuselect '^[[Z' reverse-menu-complete
+
 # -----------------------------
 # Zsh plugins (sourced directly)
 # -----------------------------
